src/analytics/formErrorTracker/formErrorTracker.test.ts

```diff
@@ -79,4 +79,49 @@ describe("FormErrorTracker", () => {
     const form = document.forms[0];
     expect(instance.getType(form)).toEqual("checkbox");
   });
+
+  test("getType with text field should return the type free text field", () => {
+    const instance = new FormErrorTracker();
+
+    document.body.innerHTML =
+      '<form action="/test-url" method="post">' +
+      '  <label for="username">test label username</label>' +
+      '  <p id="organisationType-error" class="govuk-error-message"><span class="govuk-visually-hidden">Error:</span> Select one option</p>' +
+      '  <input type="text" id="username" name="username" value="test value"/>' +
+      '  <button id="button" type="submit">submit</button>' +
+      "</form>";
+
+    const form = document.forms[0];
+    expect(instance.getType(form)).toEqual("free text field");
+  });
+
+  test("getType with textarea field should return the type free text field", () => {
+    const instance = new FormErrorTracker();
+
+    document.body.innerHTML =
+      '<form action="/test-url" method="post">' +
+      '  <label for="username">test label username</label>' +
+      '  <p id="organisationType-error" class="govuk-error-message"><span class="govuk-visually-hidden">Error:</span> Select one option</p>' +
+      '  <textarea id="username" name="username" value="test value"/>test value</textarea>' +
+      '  <button id="button" type="submit">submit</button>' +
```

Comment:
Nice range of tests

Reply:
Do we need one for radios?

Reply:
No need because radio type has no need to be overwrite (like checkbox). So the function returns directly the type define within the field element.

This scenario is already managed within the test for the checkbox.